services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_api/_volume.py

```diff
@@ -22,7 +23,9 @@
 )
 from ._utils import docker_client
 
-log = logging.getLogger(__name__)
+_logger = logging.getLogger(__name__)
+
+DIND_VERSION: Final[DockerVersion] = parse_obj_as(DockerVersion, "20.10.14")
```

Comment:
DIND??

Reply:
`dind` stays for docker in docker. I have moved this to a different module where it has more context